OsmAnd/res/values/strings.xml

```diff
@@ -3192,7 +3192,7 @@
     <string name="search_villages_and_postcodes">Search for more villages/postcodes</string>
     <string name="rendering_attr_showRoadMaps_description">Choose when to display roads-only maps:</string>
     <string name="rendering_attr_showRoadMaps_name">Roads-only maps</string>
-    <string name="safe_mode_description">Run the app in safe mode (using slower Android instead of native code).</string>
+    <string name="safe_mode_description">Run the app in safe mode (using slower Android code).</string>
```

Comment:
No. Why are you removing the "native" information?

PS: If you keep making introducing new changes to this PR while it is being reviewed, we will never get to the point of merging anything ... ;) If you are willing to reduce the PR to everything I have OK'ed, we could at least merge that and start the other stuff fresh in an new PR?

Reply:
Because it doesn't add any information the user needs.
"Run the app in (slower) safe mode" is the better alternative.

Reply:
```suggestion
    <string name="safe_mode_description">Run the app in (slower) safe mode.</string>
```